src/main/java/openblocks/common/entity/EntityHangGlider.java

```diff
@@ -48,7 +47,7 @@ public class EntityHangGlider extends Entity implements IEntityAdditionalSpawnDa
 
     private static final int PROPERTY_DEPLOYED = 17;
 
-    private static Map<EntityPlayer, EntityHangGlider> gliderMap = new MapMaker().weakKeys().weakValues().makeMap();
+    private static HashMap<EntityPlayer, EntityHangGlider> gliderMap = new HashMap<EntityPlayer, EntityHangGlider>();
```

Comment:
nitpick: The field could keep the Map type, and some type arguments are redundant.
```suggestion
    private static Map<EntityPlayer, EntityHangGlider> gliderMap = new HashMap<>();
```